frontend/src/Components/Helpers/helpers.js

```diff
@@ -0,0 +1,66 @@
+// Helper functions
+
+export function processResults(resultArr) {
+    const data_to_visualize_all = [];
+    let isTimeSeries = false;
+
+    if (resultArr[0][0] != undefined && resultArr[0][0].length === 0) {
+        isTimeSeries = false;
+    } else if (resultArr.length == 0) {
```

Comment:
If this is ever true then that means the previous conditional threw an exception that isn't being handled. Swap these around.